docker-compose.build.yml

```diff
@@ -7,6 +7,7 @@ services:
       context: .
       args:
         - NEXT_PUBLIC_LANGFUSE_CLOUD_REGION=${NEXT_PUBLIC_LANGFUSE_CLOUD_REGION}
+        - PORTER_NEXT_PUBLIC_BUILD_ID=${PORTER_NEXT_PUBLIC_BUILD_ID}
```

Comment:
we cannot use the porter arg outside of our internal (cloud) deployment pipeline

I think for docker compose we can omit this variable as it is optional